src/pipecat/transports/services/daily.py

```diff
@@ -839,6 +838,13 @@ def __init__(self, client: DailyTransportClient, params: DailyParams, **kwargs):
     def vad_analyzer(self) -> Optional[VADAnalyzer]:
         return self._vad_analyzer
 
+    def start_audio_in_streaming(self):
+        logger.debug(f"Start receiving audio")
```

Comment:
Should we move the log line inside the `if` below in case audio in is disabled?

Reply:
Done!